sas-charleston.md

```diff
@@ -0,0 +1,9 @@
+# Charleston, SC
+
+### Restaurants
+
+### Local Attractions
+  - Battery Park
+  - Folly  Beach
```

Comment:
@sherylsabol I haven't been there before! But I am from the East coast -- anything I should know before I go? 🌊